notebooks/ICESat-2_MODIS_Arctic_Sea_Ice/Customize and Access Data.ipynb

```diff
@@ -40,6 +40,17 @@
     "import requests\n",
```

Comment:
<p>Remove this sentence:</p><p> </p><p>The&nbsp;<strong>Data access for all datasets</strong>&nbsp;notebook provides the steps needed to subset and download all the data we'll be using in the final&nbsp;<strong>Visualize and Analyze Data</strong>.</p><p> </p><p>And we can update the landing page guidance to:</p><p> </p><p>There are other discovery and access methods available from NSIDC listed on the data set landing page (e.g. <a href="https://nsidc.org/data/atl07/#anchor-1" target="_blank">ATL07 Data Access &amp; Tools</a>) and <a href="https://search.earthdata.nasa.gov/" target="_blank">NASA Earthdata Search</a>. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>